pkg/virt-launcher/virtwrap/device/hostdevice/sriov/pcipool_netstatus.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package sriov
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type PCIAddressWithNetworkStatusPool struct {
+	networkPCIMap map[string]string
+}
```

Comment:
Just an observation: This could have just been `type PCIAddressWithNetworkStatusPool map[string]string`.

Reply:
and an interesting one at that.